bench/createMultipartBufferForEncodingBench.js

```diff
@@ -1,5 +1,5 @@
 function createMultipartBufferForEncodingBench(boundary, amount, charset) {
-  const filename = charset === 'utf-8' ? 'utf-8\'\'%c2%a3%20and%20%e2%82%ac%20rates' : 'iso-8859-1\'en\'%A3%20rates';
+  const filename = charset === 'utf-8' ? 'utf-8\'\'%c2%a3%20and%20%e2%82%ac%20rates' : `${charset}\'en\'%A3%20rates`;
```

Comment:
not sure if this was a right change, in this case `iso-8859-1` might actually be accurate, since I assume this is not node-specific but should be according to spec.

Reply:
I usually patch decodeText in utils.js with logging to check if the text is actually what I expect and the destencoding is not utf-8.